### ISSUE TITLE
conflicts with gftables and singular

### ISSUE THREAD

**Issue by mikestillman** [Macaulay2/homebrew-tap#97]
@mahrud I tried installing macaulay2 via homebrew on the mac, and have hit a few build issues.  What is the best way to handle this error?

```
bash-3.2$ brew tap Macaulay2/tap
bash-3.2$ brew install --only-dependencies macaulay2/tap/M2
==> Installing macaulay2 from macaulay2/tap
==> Downloading https://github.com/Macaulay2/homebrew-tap/releases/download/factory-4.2.0_1/factory-4.2.0_1.catalina.bottle.tar.gz
==> Downloading from https://github-releases.githubusercontent.com/326525187/4c94d900-d52d-11eb-81fc-0ea1f80b1a53?X-Amz-Algorithm=AWS4-HMAC-S
###################################################################################################################################### 100.0%
==> Installing dependencies for macaulay2/tap/macaulay2: factory
==> Installing macaulay2/tap/macaulay2 dependency: factory
==> Pouring factory-4.2.0_1.catalina.bottle.tar.gz
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink include/factory/cf_gmp.h
Target /usr/local/include/factory/cf_gmp.h
is a symlink belonging to singular. You can unlink it:
  brew unlink singular

To force the link and overwrite all conflicting files:
  brew link --overwrite factory

To list all files that would be deleted:
  brew link --overwrite --dry-run factory

Possible conflicting files are:
/usr/local/include/factory/cf_gmp.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/cf_gmp.h
/usr/local/include/factory/factory.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/factory.h
/usr/local/include/factory/factoryconf.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/factoryconf.h
/usr/local/include/factory/globaldefs.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/globaldefs.h
/usr/local/include/factory/si_log2.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/si_log2.h
/usr/local/include/factory/templates/ftmpl_afactor.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/templates/ftmpl_afactor.h
/usr/local/include/factory/templates/ftmpl_array.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/templates/ftmpl_array.h
/usr/local/include/factory/templates/ftmpl_factor.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/templates/ftmpl_factor.h
/usr/local/include/factory/templates/ftmpl_list.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/templates/ftmpl_list.h
/usr/local/include/factory/templates/ftmpl_matrix.h -> /usr/local/Cellar/singular/4.2.0_1/include/factory/templates/ftmpl_matrix.h
Error: Could not symlink share/factory/gftables/10201
Target /usr/local/share/factory/gftables/10201
is a symlink belonging to singular. You can unlink it:
  brew unlink singular

To force the link and overwrite all conflicting files:
  brew link --overwrite singular

To list all files that would be deleted:
  brew link --overwrite --dry-run singular
```

**Comment by mahrud** [comment]
Did you get either of the suggestions?

**Comment by mikestillman** [comment]
Not yet.  Wanted to know which you recommend.  But I can try them.

**Comment by mikestillman** [comment]
I have had further problems.  This all started when I did `brew install Macaulay2/tap/M2`

First it seemed to do a large amount of updating, including things I was surprised had anything to do with M2 (perhaps it forced a brew upgrade).  Then macaulay2 could not link libntl, as the version in homebrew was one larger than the one macaulay2 wanted.  I tried a couple of things, which I don't recall now, but including a reinstall macaulay2...  Anyway, then I ended up with the above error...
